product_pricelist_alternative/views/product_pricelist_view.xml

```diff
@@ -34,13 +34,7 @@
                 </group>
             </group>
             <xpath
-                expr="//page[@name='pricelist_rules']/field[@name='item_ids']/tree/field[@name='product_id']"
-                position="after"
-            >
-                <field name="alternative_pricelist_policy" />
-            </xpath>
-            <xpath
-                expr="//page[@name='pricelist_rules']/field[@name='item_ids']/tree[@groups='product.group_sale_pricelist']/field[@name='name']"
```

Comment:
note: `groups="product.group_sale_pricelist"` is [removed in 18.0](https://github.com/odoo/odoo/blob/18.0/addons/product/views/product_pricelist_views.xml#L74-L87).
Only a single `<list>` remain whereas [before two <tree> elements existed](https://github.com/odoo/odoo/blob/16.0/addons/product/views/product_pricelist_views.xml#L75-L107)